AssetManagementTestProject/TestCases/LoginTest.cs

```diff
@@ -1,33 +1,11 @@
-﻿using AssetManagementTestProject.PageObj;
-using AssetManagementTestProject.TestSetup;
+﻿using AssetManagementTestProject.TestSetup;
 using NUnit.Framework;
 
 namespace AssetManagementTestProject;
 
 [TestFixture]
 public class LoginTest : NUnitWebTestSetup
```

Comment:
You don't have to do the login test, so why did you keep this file? Please remove.

Reply:
Done